proxy/src/main/java/io/openlineage/proxy/ProxyStreamFactory.java

```diff
@@ -0,0 +1,42 @@
+package io.openlineage.proxy;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
+import io.openlineage.proxy.api.models.ConsoleConfig;
+import io.openlineage.proxy.api.models.ConsoleLineageStream;
+import io.openlineage.proxy.api.models.KafkaConfig;
+import io.openlineage.proxy.api.models.KafkaLineageStream;
+import io.openlineage.proxy.api.models.LineageStream;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public final class ProxyStreamFactory {
+  private static final String DEFAULT_PROXY_LINEAGE_SOURCE = "openLineageProxyBackend";
+  private static final Set<ProxyStreamConfig> DEFAULT_STREAMS =
+      Sets.newHashSet(new ConsoleConfig());
+
+  @Getter @Setter private String source = DEFAULT_PROXY_LINEAGE_SOURCE;
+  @Getter @Setter private Set<ProxyStreamConfig> streams = DEFAULT_STREAMS;
+
+  public ImmutableSet<LineageStream> build() {
```

Comment:
I'd make this a `List` rather than `Set`, just so users have the ability to set the order of the target systems this broker is sending messages to. E.g., I may want to make sure the Kafka system always gets the messages first, in case a subsequent recipient is down or sending fails for some reason.

Reply:
Valid point 👍